version.env

```diff
@@ -0,0 +1,10 @@
+#We use semantic versioning:
+#version x.y.z
+#x is an api version and if breaking api changes are introduced this should change, otherwise it remains constant
+#y is a new feature version and this only changes if new features are changed but bugs are not fixed
+#z is a patch version that changes if a bug is fixed for a previous feature version.
+# See https://semver.org/ for more information
+
+#To use this file call `source version.env` from a bash terminal
+
+export BITBUCKET_PROVIDER_VERSION=1.4.0
```

Comment:
Version is controlled by the build number/tag in Travis, please remove

Reply:
How can we retrieve the version number for our forked repositories when we are building and testing the provisioner, is there a file that Travis uses? If not it would be nice to have a way to synchronize all builds of this project.  This idea has been borrowed from maven dependency versioning which has stood the test of time.

Reply:
The travis job will publish a new release against the releases page for each tag; so the versioning is not required until it's actually merged to master and included in a release. + we are in golang-land not maven hah

You can use a script [similar to this](https://gavinbunney.github.io/terraform-provider-kubectl/docs/provider.html#install-latest-version) to fetch the latest version available

Reply:
We have removed  version.env and updated the PR.

Reply:
@gavinbunney can you explain in more detail how you're recommending the version be retrieved? Thanks

Reply:
I'm currently using `curl -Ls https://api.github.com/repos/gavinbunney/terraform-provider-bitbucketserver/releases/latest | jq -r ".tag_name"`

Reply:
This script fetches and installs the latest version: https://gavinbunney.github.io/terraform-provider-bitbucketserver/docs/provider.html#install-latest-version